SPECS/xapi-test-utils.spec

```diff
@@ -0,0 +1,49 @@
+Name:           xapi-test-utils
+Version:        0.9.0
```

Comment:
Right, if everyone believes that 1.0.0 is the right thing, then I'll change it...